src/drive/styles/dropzone.styl

```diff
@@ -0,0 +1,51 @@
+@require 'settings/z-index.styl'
+
+.fil-dropzone-teaser
+    position  absolute
+    z-index   $overlay-index
+    left      50%
+    transform translateX(-50%)
```

Comment:
Maybe it's just me, but I find this way of centering content very weird. I believe this would be more standard:

```
left 0
right 0
```

and no transform. Works at least on Chrome.
@GoOz what do you think?

Reply:
No it's good like this. `left 50%` means the item's left border is placed at 50% of its parent's width and the `translateX(-50%)` moves back the item to the left at half its own size which results on a perfect centering.
This is a common technique, I use it a lot.
`right 0 left 0` wouldn't center it at all actually, it would make the item have a full width.

Reply:
Well `right 0 left 0` def. works, though I thought it needed a fixed width, but apparently not.

But ok, cool if it's a legit way to do it, please ignore my comment!